tmt/utils.py

```diff
@@ -130,7 +130,7 @@ def debug(self, key, value=None, color=None, shift=1):
         if self.opt('debug'):
             echo(self._indent(key, value, color, shift))
 
-    def _run(self, command, cwd):
+    def _run(self, command, cwd, log):
```

Comment:
Ah, I was gonna fix #48, but as you're touching it: We need to add `shell=True`. Without it we can't handle pipes, output redirection etc.

```
subprocess.Popen( . . . , shell=True)
```

Reply:
Not sure whether it should be default, but for `execute` it has to be, as Vagrant's execute is, and there's no easy way to change it.

Reply:
ack, will add it

Reply:
@pvalena added, sorry for the delay!